standard/expressions.md

```diff
@@ -4811,24 +4808,14 @@ It is possible for anonymous function delegates to share some captured variables
 > }
 > ```
 >
-> the three delegates capture the same instance of `x` but separate instances of `y`, and the output is:
->
-> ```console
-> 1 1
-> 2 1
-> 3 1
-> ```
->
-> *end example*
+> the three delegates capture the same instance of `x` but separate instances of `y`. *end example*
```

Comment:
Why was this change made? I really think we want the output here - and for it to be tested.

Reply:
@jskeet As the example was written, while it captured the delegates in an array of 3 elements, it never actually executed those functions, so no output was produced. What I will do in a separate PR is to add a loop that executes the functions, so output is actually produced, and we can check that.

Reply:
Sounds good to me.